src/design-guidelines/boolean/examples/radio-buttons.tsx

```diff
@@ -23,19 +23,6 @@ export class BooleanRadioButtonsExample {
         { text: 'Only myself', value: 2, selected: false },
     ];
 
-    @State()
-    private selectedItem1: ListItem | ListSeparator;
-    private selectedItem2: ListItem | ListSeparator;
```

Comment:
@Kiarokh I removed these since they were not used and caused the build to fail when building the docs
Please check if you forgot something or fix the example :smile:

Reply:
https://github.com/Lundalogik/lime-elements/assets/35954987/90637e6c-eedd-4d3d-af6f-6689231cc4e3

Without those, this will happen. 😓

Reply:
Well it was broken before as well :smile:

Reply:
> Without those, this will happen. 😓 

Hmm… I assume that means that adding `selectedItem1` and `selectedItem2` solved that problem with the radio-buttons? 😲 🤨